src/fielddef.ts

```diff
@@ -20,6 +20,8 @@ export interface FieldRefOption {
   binSuffix?: string;
   /** append suffix to the field ref (general) */
   suffix?: string;
+  /** return the base field, not rank */
+  noRank?: boolean;
```

Comment:
But when people read the code for `model.field`, they would not expect that color returns `rank`.   If we use `prefn:` then it is explicit that in that case it returns something different.   And in the normal case (despite used less frequently), it returns the field normally.